builtin/providers/aws/data_source_aws_ecs_container_definitions_model.go

```diff
@@ -0,0 +1,18 @@
+package aws
+
+type ecsContainerDefinition struct {
```

Comment:
Is there any specific reason we can't use the official structs from the SDK? https://github.com/aws/aws-sdk-go/blob/master/service/ecs/api.go#L3135